PhysicsTools/PatAlgos/python/slimming/applyDeepBtagging_cff.py

```diff
@@ -60,8 +60,18 @@ def applyDeepBtagging( process, postfix="" ) :
        rParam = 0.8,
        jetCorrections = ('AK8PFPuppi', cms.vstring(['L2Relative', 'L3Absolute']), 'None'),
        btagDiscriminators = [
-          'pfDeepDoubleBJetTags:probQ',
-          'pfDeepDoubleBJetTags:probH',
+          'pfDeepDoubleBvLJetTags:probQCD',
+          'pfDeepDoubleBvLJetTags:probHbb',
+          'pfDeepDoubleCvLJetTags:probQCD',
+          'pfDeepDoubleCvLJetTags:probHcc',
+          'pfDeepDoubleCvBJetTags:probHbb',
+          'pfDeepDoubleCvBJetTags:probHcc',
+          'pfMassIndependentDeepDoubleBvLJetTags:probQCD',
+          'pfMassIndependentDeepDoubleBvLJetTags:probHcc',
```

Comment:
this looks like a typo: BvL has only probQCD and probHbb options

Reply:
Thanks for catching that